.circleci/config.yml

```diff
@@ -76,9 +87,16 @@ workflows:
           filters:
             branches:
               only: master
+      - docs:
+          requires:
+            - test
+          filters:
+            branches:
+              only: master
       - release:
           requires:
             - build
+            - docs
```

Comment:
This flow matches the components flow: https://github.com/webex/components/blob/master/.circleci/config.yml#L101